lang/python/setup_pip.py

```diff
@@ -231,7 +231,8 @@ def get_library_dirs():
 inc_paths = [ os.path.join(build_dir, 'include'), os.path.join(build_dir, 'config'), build_dir, '.' ]
 lib_paths = [ '.' ]
 
-check_needed_dependencies(builtins, inc_paths, lib_paths)
+# Find and locate dependencies to be linked - optional
+# check_needed_dependencies(builtins, inc_paths, lib_paths)
```

Comment:
Remove the commented lines.

Reply:
removed in most recent commit: [924e8e7](https://github.com/wiredtiger/wiredtiger/pull/10492/commits/924e8e795d33874fa216d570580193aa91958644)